main.go

```diff
@@ -67,6 +67,7 @@ func main() {
 		logLevel      = flag.String("log.level", "INFO", "Log level to log from (DEBUG|INFO|WARN|ERROR).")
 		promPort      = flag.Int("prom.port", 10100, "Port to expose prometheus metrics.")
 		cacheDuration = flag.Duration("cache.duration", 300*time.Second, "Cache expiry time.")
+		collectUsage  = flag.Bool("collect.usage", false, "Collect quotas usage where available (NOTE: CloudWatch calls aren't free)")
```

Comment:
I'm not sure what is preferred, so far made it disabled by default.
Can remove completely and keep always enabled.